README.md

```diff
@@ -1,2 +1,401 @@
-# example-application
-Example out-of-tree application that is also a module
+![Zephyr Project logo](https://github.com/zephyrproject-rtos/zephyr/blob/v1.14-branch/doc/images/Zephyr-Project.png)
+
+
+# Module Overview
+
+The Zephyr Project is a scalable real-time operating system (RTOS) supporting
+multiple hardware architectures, optimized for resource constrained devices,
+and built with security in mind.
+
+The Zephyr build system supports code tree "plug-ins" called *modules*.
+See the [Zephyr Modules (External projects)][5] for a high-level overview
+of the build system module mechanism.
+
+
+## Scope
+
+The Zephyr eXample Application (ZXA) module is supplementary to the Zephyr
+repository.  This provides:
+
+1. A reference implementation of using the Zephyr module mechanism.
+
+1. A repository for holding example code and config files outside of the
+   Zephyr repository, including:
+
+   - `west` manifest
+   - projects/samples/tests
+   - applications/subsystems/drivers/interface declarations
+   - device tree config files
+   - boards
+   - scripts
+
+1. A test case for CI to monitor Zephyr build system support for modules.
+
+
+   _WARNING:_ The Zephyr repository is intentionally
+   **independent of and unaware of this repository**.
+
+
+# Getting Started
+
+## Creating a New Module
+
+This module is designed to be a boilerplate for other modules simply by:
+
+1. cloning
+
+1. renaming module names and abbreviations to reference your module:
+
+  - 'eXample Application'
+  - 'example-application'
+  - 'ZXA'
+  - 'zxa'
+
+1. stripping out any unnecessary content (e.g. no drivers or subsystems)
+
+  - If you don't have any non-sample, non-test projects:
+
+    1. remove the `zephyr/projects/` subdirectory
+    1. In `zephyr/modules.yml` remove the reference to `zephyr/projects`
+
+
+## Creating a Free-Standing Manifest Repository
+
+This example module is also an appropriate template for creating a
+manifest-only repository because, like manifest modules, they may
```

Comment:
I am not sure what a "manifest module" is, I would remove this.

Reply:
> manifest module

Agree that this is mixing terms that apply to different layers in the software stack.